src/index.html

```diff
@@ -1,20 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-  </head>
-  <body>
-    <div>
-      <h2 class="bg-primary grid">main section</h2>
-      <h2>main section</h2>
-      <!-- Banner section -->
-      <section class="banner font-bold">
-        <h2>banner titile</h2>
-        <p>banner description</p>
-      </section>
-      <h3>Test 2</h3>
-    </div>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+</head>
+
+<body>
+  <div>
+    <h2 class="bg-primary grid">main section</h2>
+    <h2>main section</h2>
+    <!-- Banner section -->
+    <section class="banner font-bold">
+      <h2>banner titile</h2>
+      <p>banner description</p>
+    </section>
+    <h3>Test 2</h3>
+    <section class="my-8 py-4 bg-grey-bg flex justify-center items-center">
+      <button class='flex justify-center items-center max-w-xs w-full h-14 border-2 border-dashed border-accent bg-grey-shd7 rounded-md text-accent hover:bg-grey-shd6 focus:outline-none'>
```

Comment:
the button has so many classes, maybe this is a good place to start using the @apply directive

Reply:
I think if we were to write vanilla CSS with vanilla JS, I would have made use of the @apply directive to reuse the CSS classes. However, since we will be using re-usable react components, I see there is no need to re-use CSS classes also. Nevertheless, it can be used as a syntactic sugar here.

Reply:
it is not a syntactic sugar, ut is needed to be separate classes without `@apply` directive as those classes will be switched and toggled to achieve all desired look and feels in the future,otherwise you would right one `@apply` directive for different button cases.

Reply:
@Nilegfx I see your point here. I have changed it accordingly.

Reply:
@Safei-Ashraf the @apply directive was used as described.

Reply:
I think you got me wrong.

I meant NOT to use the apply directive and keep the classes on the component for easier class toggling and switching.

So I support your original argument @Sahl-Khalifa

Reply:
Thank you for the correction. The classes were modified as advised.